src/main/clm_varpar.F90

```diff
@@ -63,9 +68,53 @@ module clm_varpar
   integer, parameter :: i_cel_lit  = i_met_lit + 1
   integer, parameter :: i_lig_lit  = i_cel_lit + 1
   integer            :: i_cwd
+   !Matrix index (when use_matrixcn)
+  integer, parameter :: ileaf         = 1
```

Comment:
These have good names, but I think it should still have comments to describe what they are.

Reply:
Still needs to be done

Reply:
done

Reply:
done